states/massachusetts/elections.toml

```diff
@@ -9,41 +9,44 @@ name = "Congressional Primary"
 original_date = 2020-09-01
 
 [20200901.registration]
+# Note the day to register was extended to 10 days from 20 days prior in 2020
 in_person_by = 2020-08-22
 online_by = 2020-08-22T23:59:00
 postmarked_by = 2020-08-22
 sources = [
   "http://www.sec.state.ma.us/ele/eleifv/howreg.htm",
   "https://www.sec.state.ma.us/ele/elepdf/2020-Primary-and-Election-Schedule-Legal-Sized.pdf",
 ]
-# Note the Day to Register was extended to 10 days from 20 days prior in 2020
 
-[20200901.absentee.application]
-email_by = 2020-10-31T12:00:00
-fax_by = 2020-10-31T12:00:00
-received_by = 2020-10-31T12:00:00
-# Because you can vote absentee in-person day before is OK
-sources = ["http://www.sec.state.ma.us/ele/eleabsentee/absidx.htm"]
+[20200901.emergency.application] # This is true absentee
+email_by = 2020-08-18T12:00:00 # Padded 14 days
+fax_by = 2020-08-18T12:00:00 # Padded 14 days
+received_by = 2020-08-18T12:00:00 # Padded 14 days
+sources = ["http://www.sec.state.ma.us/ele/eleabsentee/absidx.htm"] 
```

Comment:
Don't pad these dates:

> Emergency Ballots
>
> If you have been admitted to a healthcare facility within 7 days of an election, or you have been instructed by a medical professional or public health official to self-quarantine in your home within 7 days of an election, you may use the absentee ballot application to designate someone to pick up your emergency absentee ballot, deliver it to you, and return it to your local election official.
>
> There is no deadline to submit an application for an emergency ballot, but your ballot must be returned to your local election office by the close of polls.

From: http://www.sec.state.ma.us/ele/covid-19/covid-19.htm

Reply:
Because we're merging concepts we're going to be imprecised for some cases, I'll unpad and add back an in_person.

http://www.sec.state.ma.us/ele/eleabsentee/absidx.htm 

"Absentee ballot applications must be received by your local election office by 12 p.m. on the day before the election, unless the day before the election is a holiday or a Sunday, in which case the application must be submitted by 5 p.m. on the last business day before the election.

Applications may be submitted in person, by mail, fax, or by e-mail; however, the local election official must be able to view the signature of the person requesting the ballot. If the application is being submitted by e-mail, a scanned copy of the request must be submitted."